src/Dropdown/Dropdown.tsx

```diff
@@ -14,12 +14,14 @@ export interface DropdownProps {
   onChange: ChangeEventHandler;
   /** The name of the dropdown */
   name: string;
-  /** An array of objects with the properties 'label' and 'value' for each dropdown option */
-  options: Array<{label: string; value: string}>;
+  /** An array of objects with the properties 'label', 'value', and 'disabled' for each dropdown option */
+  options: Array<{label: string; value: string; disabled?: boolean}>;
```

Comment:
These additional options could do with having their own test cases. For example:

- What happens if I set `options.disabled` to `true`? 
- What happens if I set `options.disabled` to `false`
- What happens if I don't provide a value for `options.disabled`?

Reply:
Good point! I added tests through 03a9141.